### PR TITLE
Converts slider inputs to radio buttons

### LINKED ISSUE #96: [BUG] UX: subjective page should not select slider values by default.
**Describe the bug**
On the subjective page, by default, the sliders are selected with the lowest value. User might hit continue quickly and miss this page. It might be better to force user select a value.

**To Reproduce**
1. Go to 'home'
1. Click on 'Check up on your health'
1. Click on 'Start checkup'
1. See that the sliders are selected

**Expected behavior**
We should force user to select a value so they don't miss this page.

**Screenshots**
Screen Shot 2020-03-23 at 6 36 27 PM

related to https://github.com/coronavirus-diary/coronavirus-diary/issues/82

### PR DESCRIPTION
**Pull request by gspencergoog**
Because we'd like to be able to not bias the inputs, the form should have "no value" for each question initially. Sliders don't really lend themselves to that.

This changes the sliders out for radio boxes.

In addition, now that we can have "no value" entries, I made moving on to the next screen disabled unless all the questions have values.

I did change the data format for the questions, so perhaps that is not OK, but it appears to send the same data to the server, so maybe it is.

I also adjusted the temperature page to not jump around so much when entering values, by placing a `minHeight` constraint on it. And I removed the little thermometer icon.

Fixes #96

Here's what the subjective page looks like now.

image

And the temperature page:
image

### CODE REVIEW

File: lib/src/data/repositories/questions.dart [hunk 1]
```diff
@@ -8,36 +8,59 @@ class QuestionsRepository {
...
-            '0': "I can breathe fine",
-            '4': "I can't breathe at all",
-          },
+          initialValue: null,
+          labels: [
+            'I can breathe fine',
+            '',
+            '',
+            '',
+            "I can hardly breathe at all",
```
Comment (dnfield) [inline lib/src/data/repositories/questions.dart:17]: Can we move these to localization?  As well as below and semantics labels.

Comment (gspencergoog) [reply lib/src/data/repositories/questions.dart:17]: Yes, I was thinking about that last night.  OK, I'll do that.

Comment (clocksmith) [reply lib/src/data/repositories/questions.dart:17]: I started adding them here: https://github.com/coronavirus-diary/coronavirus-diary/compare/develop...clocksmith:feature/127#diff-f1fa22d514621ca64fd2e088a83a5876

per https://github.com/coronavirus-diary/coronavirus-diary/issues/127

Comment (gspencergoog) [reply lib/src/data/repositories/questions.dart:17]: I don't see anything at that link ("no differences").

I'm happy to let you do it, I'm having trouble getting an instance of AppLocalizations into the Bloc data anyhow.

@dnfield OK if we defer that localization to another PR?

Comment (clocksmith) [reply lib/src/data/repositories/questions.dart:17]: sorry here is the correct link: https://github.com/coronavirus-diary/coronavirus-diary/compare/develop...clocksmith:feature/127

Comment (gspencergoog) [reply lib/src/data/repositories/questions.dart:17]: Oh, I see. Yeah, I got that far too, but ran into trouble when I wanted to access the AppLocalizations before the MaterialApp had been created. Not quite sure how to solve that yet.

Comment (dnfield) [reply lib/src/data/repositories/questions.dart:17]: Ahhh.  Perhaps we could set up the localizations delegate in main, and pass that to the bloc supervisor as well as to the MaterialApp.  But yeah, putting that in a separate PR is fine.

Comment (dnfield) [reply lib/src/data/repositories/questions.dart:17]: Filed https://github.com/coronavirus-diary/coronavirus-diary/issues/134

**Comment by gspencergoog** [comment]
@dnfield I can haz LGTM?

**Review by dnfield — APPROVED** [review]
LGTM!

Sorry forgot to click the button after opening this haha

**Pull request merged**